tests/ceph_volume/test_ceph_volume_lvm_tags_after_osd_rm.py

```diff
@@ -0,0 +1,74 @@
+from json import loads
+
+from ceph.waiter import WaitUntil
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.utils import get_lvm_list, get_running_containers
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class FailedToRemoveOsdError(Exception):
+    pass
+
+
+def run(ceph_cluster, **kw):
+    """Check LVM tags post removing OSD (--zap)
+    Args:
+        **kw: Key/value pairs of configuration information to be used in the test.
+    """
+    osd_node = ceph_cluster.get_nodes(role="osd")[0]
+
+    # Get the lvm list before osd remove
+    running_containers, out = get_running_containers(
+        osd_node, format="json", expr="name=osd", sudo=True
+    )
+    container_ids = [item["Names"][0] for item in loads(running_containers)]
+    lvm_list = get_lvm_list(container_ids[0], osd_node)
+
+    # Identify an OSD ID to perform
+    osd_id = list(lvm_list.keys())[0]
+
+    # Change the replication to 2 / default is 3
+    out = CephAdm(osd_node).ceph.osd(command="pool ls detail", format="json")
```

Comment:
Is this any general command or specific commands ?

Reply:
It is an osd specific command Vaibhav